src/runtime_src/core/common/info_aie.cpp

```diff
@@ -271,7 +271,16 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
     std::string aie_data = xrt_core::device_query<qr::aie_shim_info>(device);
     std::stringstream ss(aie_data);
     boost::property_tree::read_json(ss, pt_shim);
-  } catch (const std::exception& ex){
+  }
+  catch(const xrt_core::query::no_such_key&) {
+    pt.put("error_msg", "AIE information is not available");
+	return pt;
+  }
+  catch(const xrt_core::query::exception&) {
+	pt.put("error_msg", "AIE information is not available");
+	return pt;
+  }
+  catch (const std::exception& ex){
     pt.put("error_msg", ex.what());
     return pt;
```

Comment:
I think there is a mix of tabs a spaces here, can you untabify and line up?
Please consistently add space between stmt and condition as in space between catch and parent `catch (`
 and space between condition and curly brace also.

Reply:
fixed.